frontend/src/app/offers/create-offer/create-offer.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, OnInit } from '@angular/core';
+import { Offer } from 'app/homepage-offer/offers.model';
+import { AuthService } from 'app/auth.service';
+import { User } from 'app/user';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import { OffersService } from 'app/homepage-offer/offers.service';
+import { ActivatedRoute, Params } from '@angular/router';
+import { environment } from 'environments/environment';
+
+@Component({
+  selector: 'volontulo-create-offer',
+  templateUrl: './create-offer.component.html',
+  styleUrls: ['./create-offer.component.scss']
+})
+export class CreateOfferComponent implements OnInit {
+  djangoRoot: string;
+  public inEditMode: boolean = false;
+  offer: Offer = new Offer;
```

Comment:
what is this `new Offer` supposed to do?

Reply:
If I delete it I cannot display list of user's organizations while creating offer. Being honest I don't understand why... I still can console.log it then, but it don't appear in select options.